Bio/File.py

```diff
@@ -183,19 +191,24 @@ def peekline(self):
         return line
 
     def tell(self):
+        """Return the current position of the file read/write pointer within the File."""
         return self._handle.tell() - sum(len(line) for line in self._saved)
 
     def seek(self, *args):
+        """Set the current position at the offset specified."""
         self._saved = []
         self._handle.seek(*args)
 
     def __getattr__(self, attr):
+        """Return File attribute."""
         return getattr(self._handle, attr)
 
     def __enter__(self):
+        """Open File handle with WITH statement."""
```

Comment:
Repetitive use of the word 'with' here and in ``__exit__`` (same in ``bgzf.py`` but I didn't notice till now). How about:

```python
"""Special method called when opening the file using a with-statement."""
```

and

```python
"""Special method called when closing the file using a with-statement."""
```